src/iexamine.cpp

```diff
@@ -1133,18 +1137,12 @@ void iexamine::gunsafe_el(player &p, const tripoint &examp)
 }
 
 /**
- * Checks that player is outside and has crowbar then calls iuse.crowbar.
+ * Checks PC has a crowbar then calls iuse.crowbar.
```

Comment:
technically, it's still a player and could be called by an NPC.

Reply:
Fair enough, didn't think NPC's could examine things.